lib/ofex.ex

```diff
@@ -106,8 +106,16 @@ defmodule Ofex do
 
   defp parse_message_set("SIGNUPMSGSRSV1", message_set), do: SignonAccounts.create(message_set)
   defp parse_message_set("SIGNONMSGSRSV1", message_set), do: Signon.create(message_set)
-  defp parse_message_set("BANKMSGSRSV1", message_set), do: BankAccount.create(message_set)
-  defp parse_message_set("CREDITCARDMSGSRSV1", message_set), do: CreditCardAccount.create(message_set)
+  defp parse_message_set("BANKMSGSRSV1", message_set) do 
+    message_set 
+    |> xpath(~x"//BANKMSGSRSV1/*[contains(name(), 'STMTTRNRS')]"l) 
+    |> Enum.map(&BankAccount.create(&1)) 
+  end 
+  defp parse_message_set("CREDITCARDMSGSRSV1", message_set) do 
+    message_set 
+    |> xpath(~x"//CREDITCARDMSGSRSV1/*[contains(name(), 'CCSTMTTRNRS')]"l) 
```

Comment:
For these selectors, can we be explicit rather than just checkin on a `contains`?
```elixir
|> xpath(~x"//CREDITCARDMSGSRSV1/CCSTMTTRNRS"l)
```